[2.13] backport: Exclude security-webauthn from QuarkusExtensionsCombination coverage

pjgg opened the pull request:
"security-webauthn" extension is not officially supported on 2.13 release

backport: b6a5763c5ab068e4b1dab8463c455618f9c7ac18

rsvoboda reviewed (commented):
Hold on, we need to understand what's wrong.

The fact that product has no support plans for webauthn extension doesn't mean that it needs to be excluded

rsvoboda commented:
I can reproduce issue with webauthn extension
```
mvn io.quarkus.platform:quarkus-maven-plugin:2.13.3.Final:create \
-DprojectGroupId=my-groupId \
-DprojectArtifactId=my-artifactId-latest -Dextensions=security-webauthn
mvn clean verify -f my-artifactId-latest
```

Commit:
Upstream issue: Exclude security-webauthn from QuarkusExtensionsCombination coverage due to a DI error at compile time

The Default Quarkus application generated by Maven plugin with security-webauthn extension is throwing an exception on compile time.
The error is related to a dependency injection issue
Ref: https://github.com/quarkusio/quarkus/issues/28746

(cherry picked from commit b6a5763c5ab068e4b1dab8463c455618f9c7ac18)
